tests/e2e-tests/specs/backend/product-search.test.js

```diff
@@ -2,37 +2,67 @@
  * External dependencies
  */
 import {
-	insertBlock,
-	getEditedPostContent,
-	createNewPost,
 	switchUserToAdmin,
+	getEditedPostContent,
+	openDocumentSettingsSidebar,
 } from '@wordpress/e2e-test-utils';
 import { clearAndFillInput } from '@woocommerce/e2e-tests/utils';
+import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+
+/**
+ * Internal dependencies
+ */
+
+const block = {
+	name: 'Product Search',
+	slug: 'woocommerce/product-search',
+	class: '.wc-block-product-search',
+};
 
-describe( 'Product Search', () => {
-	beforeEach( async () => {
+describe( `${ block.name } Block`, () => {
+	beforeAll( async () => {
 		await switchUserToAdmin();
-		await createNewPost();
+		await visitBlockPage( `${ block.name } Block` );
 	} );
 
-	it( 'can be created', async () => {
-		await insertBlock( 'Product Search' );
+	it( 'renders without crashing', async () => {
+		// Gutenberg error
+		expect(
+			( await page.content() ).match(
+				/Your site doesn’t include support for/gi
+			)
+		).toBeNull();
+		// Our ErrorBoundary
+		expect(
+			( await page.content() ).match(
+				/There was an error whilst rendering/gi
+			)
+		).toBeNull();
+		// Validation Error
+		expect(
+			( await page.content() ).match(
+				/This block contains unexpected or invalid content/gi
+			)
+		).toBeNull();
 
-		expect( await getEditedPostContent() ).toMatchSnapshot();
+		await expect( page ).toMatchElement( block.class );
 	} );
 
 	it( 'can toggle field label', async () => {
-		await insertBlock( 'Product Search' );
+		await openDocumentSettingsSidebar();
+		// we focus on the block
+		await page.click( block.class );
```

Comment:
In future versions of GB, the settings sidebar is closed by default, see open it anyway.
also when visiting a page, no block is focused by default, so we click on our block to see the settings.